### ISSUE TITLE
I cannot comment on a specific post in my own repository for some reason: maybe OP blocked me or repo transfer bug

### ISSUE THREAD

**Issue by cirosantilli** [isaacs/github#1570]
This is the thread: https://github.com/cirosantilli/china-dictatorship/issues/42

Someone commented there 5 hours ago so it seems that I'm the only one who can't comment anymore: https://github.com/cirosantilli/china-dictatorship/issues/42#issuecomment-489290554

The comment box says: "You can't perform this action at this time."

I'm not sure what is the reason for that, but this should never happen since that is my own repository.

Two possible theories I have:

- I transferred that issue from another repo recently, and archived the original repo, and there is a bug with that. But I also transferred other issues, and I can comment on them.
- the OP has blocked me after opening the issue. But I seem to be able to create issues on OPs repositories so does not look like it (didn't actually create issues, but "Submit new issue" button does show normally in OPs repo). And even if that is the case, this is a bad implementation, I should always be able to comment on my own repos no matter what.

**Comment by georgexsh** [comment]
yeah, op has blocked me too and changed the issue title to something hilarious.

**Comment by cirosantilli** [comment]
Thanks for letting me know, updated issue now.
